WcaOnRails/db/migrate/20180623171213_add_registration_requirements_columns_to_competitions.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class AddRegistrationRequirementsColumnsToCompetitions < ActiveRecord::Migration[5.2]
+  def change
+    add_column :Competitions, :on_the_spot_registration, :boolean
+    add_column :Competitions, :on_the_spot_entry_fee_lowest_denomination, :integer, null: false, default: 0
+    add_column :Competitions, :refund_policy_percentage, :integer, null: false, default: 0
```

Comment:
nit: For consistency with [the WCIF](https://docs.google.com/document/d/1hnzAZizTH0XyGkSYe-PxFL5xpKVWl_cvSdTzlT_kAs8/edit#heading=h.nxv3vn46b4ok), I think we should call this `refund_policy_percent`, rather than `refund_policy_percentage`. As I understand it, "percent" is for a number, and "percentage" is a noun to describe a vague "piece" of something. From https://owlcation.com/humanities/Percent-vs-Percentage:

> A percent is a number; a percentage is an amount.

Reply:
I will change this.